generator/internal/sidekick/sidekick.go

```diff
@@ -17,57 +17,94 @@ package sidekick
 
 import (
 	"fmt"
-	"maps"
 	"os"
+	"strings"
+	//"maps"
+	//"os"
 )
 
+var CmdSidekick = newCommand(
+	"sidekick",
+	"sidekick is a tool for automating code generation.",
```

Comment:
s/automating code generation/automating SDK generation/ I am aiming for more than just generating the code, the goal is to generate the SDK (code, examples, documentation, README, package files, etc. etc.)